target/linux/ath79/image/generic.mk

```diff
@@ -2554,6 +2554,22 @@ define Device/teltonika_rut955-h7v3c0
 endef
 TARGET_DEVICES += teltonika_rut955-h7v3c0
 
+define Device/trendnet_tew-673gru
+  SOC := ar7161
+  DEVICE_VENDOR := Trendnet
+  DEVICE_MODEL := TEW-673GRU
+  DEVICE_VARIANT := v1.0R
+  DEVICE_PACKAGES := -uboot-envtools kmod-usb-ohci kmod-usb2 \
+  kmod-owl-loader kmod-switch-rtl8366s
+  IMAGE_SIZE := 7744k
+  FACTORY_SIZE := 6144k
+  IMAGES += factory.bin
+  IMAGE/factory.bin = append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | \
+  pad-rootfs | check-size $$$$(FACTORY_SIZE) | pad-to $$$$(FACTORY_SIZE) | \
+  append-string AP94-AR7161-RT-080619-01
+endef
```

Comment:
Please use tab instead of spaces in line 2563, 2568 and 2569.

Reply:
Ah, my editor must be messing up (aka setup wrong... user error) - I looked through and literally none of the other lines in the whole generic.mk are indented like I see on github.  I put in a tab - if it doesn't show up correctly, I'll go back and figure out how to fix it.  Thanks for catching that.

Reply:
seems still spaces, just copy one from line 2547 and paste it here. And please add one new line in the end of the dts file. It seems that you dropped two blank lines before.

Reply:
okay... that was comically bad... but should be fixed now.
Had to turn off "detect indentation" for makefile in VSCode... all better.
Of course then it was misaligned, but I think I have it fixed now...
Copying a tab wouldn't work because the editor wasn't honoring them... all looked like spaces or nothing at all.
Should be correct now... Sorry about all the force pushes... 😩

Reply:
> And please add one new line in the end of the dts file. It seems that you dropped two blank lines before.

So do not forget that😄. 
And please reorganize your commit message with `git commit --amend` and delete the useless messages after ` signed-off-by xxx`.